clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```diff
@@ -116,11 +115,8 @@ private static KeyStore createEmptyKeyStore() throws GeneralSecurityException, I
 
     private static void saveKeyStore(KeyStore ks, String filename,
                                      Password password) throws GeneralSecurityException, IOException {
-        FileOutputStream out = new FileOutputStream(filename);
-        try {
+        try (FileOutputStream out = new FileOutputStream(filename)) {
```

Comment:
We are using try with resources which closes it automatically:

https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html